packages/node/src/lib/process-request.ts

```diff
@@ -207,14 +210,16 @@ export default function processRequest(
     queryString: searchToArray(reqUrl.searchParams),
     postData,
     // TODO: When readme starts accepting these, send the correct values
-    cookies: [],
+    cookies: [] as Cookie[],
     headersSize: -1,
     bodySize: -1,
   };
 
   if (requestData.postData === null) {
-    delete requestData.postData;
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { postData: postDataToBeOmitted, ...remainingRequestData } = requestData;
```

Comment:
Do we know why `postData` has to be removed from the return instead of allowed to be null? Might be worth a comment

Reply:
yeah I'm not quite sure why the logic was this way but I wanted to confine the scope of this PR to preserve the existing logic as much as possible!

Reply:
looks like `postData` is optional! ([source](http://www.softwareishard.com/blog/har-12-spec/#request))

<img src="https://github.com/readmeio/metrics-sdks/assets/8854718/37e633b0-561a-4778-9978-02f48536eb8c" width=400>